frontend/src/services/auth/sessionUtils.ts

```diff
@@ -0,0 +1,46 @@
+import { KeyObject } from "crypto";
```

Comment:
where I ended up is a little confusing, but the idea was to split out functionality that needs access to JWT keys from functionality that doesn't in order to help with testing and organization. I think we could put everything back in one file at this point, given what I learned in the process, but leaving as is for now

Reply:
`deleteSession` seems like it belongs with `createSession` but the others make sense as `utils`. Could do as a follow-up though.